src/mlpack/methods/ann/layer/multi_layer_impl.hpp

```diff
@@ -392,7 +390,7 @@ void MultiLayer<MatType>::InitializeForwardPassMemory(const size_t batchSize)
   {
     const size_t layerOutputSize = network[i]->OutputSize();
     MakeAlias(layerOutputs[i], layerOutputMatrix, layerOutputSize, batchSize,
-        start * layerOutputMatrix.n_rows);
+        start);
```

Comment:
I'm uncertain why I included `layerOutputMatrix.n_rows` here in the first place.  It is generally 1 so it is not necessary.